src/inmanta/server/services/__init__.py

```diff
@@ -15,3 +15,6 @@
 
     Contact: code@inmanta.com
 """
+
+# This variable can be updated by the test suite to disable the scheduling of background jobs on service start.
+DONT_RUN_BACKGROUND_JOBS = False
```

Comment:
Perhaps `DISABLE_BACKGROUND_JOBS`? It's the same in essence but it feels less like a confusion double negation to me.